app/models/licence_transaction.rb

```diff
@@ -21,9 +22,14 @@ def self.title
     "Licence"
   end
 
+  def schema_organisations
+    return [] unless @creator_organisation
+
+    [@creator_organisation]
+  end
+
   def primary_publishing_organisation
-    # Set to GDS for testing
-    "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+    @creator_organisation
```

Comment:
What happens if no `primary_publishing_organisation` is set? Perhaps we should keep a default here?

Reply:
I _think_ that can't happen, because the only way you can get to document.save (where this is called) on a new licence is going through the controller's create action. 

That said, I clearly made a different decision here: https://github.com/alphagov/specialist-publisher/pull/2288/files#diff-0221e5c51589e073e450a8584ee6385eed859ac34eaaec48f886d4a2786d9f3bR25-R29 .. (although that method is returning an array, so it needs some massaging).

I think on balance it won't hurt to be cautious about a default (although I think in practise the safeguard will never be needed). I'll update.